debian/control

```diff
@@ -0,0 +1,58 @@
+Source: libisal
+Priority: optional
+Maintainer: Debian OpenStack <team+openstack@tracker.debian.org>
+Uploaders:
+ Ondřej Nový <onovy@debian.org>,
+Build-Depends:
+ debhelper (>= 12),
+ nasm
+Standards-Version: 4.3.0
+Section: libs
+Homepage: https://github.com/01org/isa-l
+Vcs-Git: https://salsa.debian.org/openstack-team/third-party/libisal.git
+Vcs-Browser: https://salsa.debian.org/openstack-team/third-party/libisal
+
+Package: isal
+Section: utils
+Architecture: amd64 kfreebsd-amd64
+Depends:
+ ${misc:Depends},
+ ${shlibs:Depends},
+Description: Intel(R) Intelligent Storage Acceleration Library - tools
+ Collection of low-level functions used in storage applications.
+ Contains fast erasure codes that implement a general
+ Reed-Solomon type encoding for blocks of data that helps protect against
+ erasure of whole blocks. The general ISA-L library contains an expanded
+ set of functions used for data protection, hashing, encryption, etc.
+ .
+ This package contains CLI tools.
+
+Package: libisal-dev
+Section: libdevel
+Architecture: amd64 kfreebsd-amd64
+Depends:
+ libisal3 (= ${binary:Version}),
+ ${misc:Depends},
+Description: Intel(R) Intelligent Storage Acceleration Library - devel files
+ Collection of low-level functions used in storage applications.
+ Contains fast erasure codes that implement a general
+ Reed-Solomon type encoding for blocks of data that helps protect against
+ erasure of whole blocks. The general ISA-L library contains an expanded
+ set of functions used for data protection, hashing, encryption, etc.
+ .
+ This package contains the development files needed to build against the shared
+ library.
+
+Package: libisal3
```

Comment:
Why are we using libisal3 when the major version appears to be 2?

```
[2019-07-18T19:41:24.079Z] -rw-r--r-- root/root    257280 2019-06-27 14:41 ./usr/lib/x86_64-linux-gnu/libisal.so.2.0.26
```

Reply:
This release removed a public symbol from the shared library with out bumping the soname,

I see though that future Ubuntu eoan code name has a 2.27.0-1 still using the libisal2 name.
Ubuntu 19.04 has has libisal2 2.25.

Reply:
The future Ubuntu 2.27.0 just removed that symbol and kept the libname the same.
It also does not flag that it breaks the older releases.

Reply:
This seems like it's an upstream issue.  Hopefully they get better with their SONAME versioning.